### PR TITLE
By default, no email status updates to users when ordering granules

### LINKED ISSUE #238: Passing the 'email' parameter to order requests should default to False.
Since the `email` parameter is optional when making orders, a flag should be added to `query.order_granules` that allows the user to opt into email notifications (the flag should be `False` by default). For users who still wish to receive email updates, the flag can be set to `True`.

NSIDC has been inundated with emails resulting from orders made by icepyx. We believe part of the problem is the small page size of 10 (see https://github.com/icesat2py/icepyx/issues/237), but making emails for orders optional will also help to alleviate (or completely eliminate) this issue.

### PR DESCRIPTION
**Pull request by trey-stafford**
resolves #238

### CODE REVIEW

**Comment by trey-stafford** [comment]
Note that the failing test appears to be an issue on the `development` branch as well.

File: examples/ICESat-2_DAAC_DataAccess_Example.ipynb [hunk 1]
```diff
@@ -427,7 +427,7 @@
    "cell_type": "markdown",
    "source": [
     "#### Place the order\n",
-    "Then, we can send the order to NSIDC using the order_granules function. Information about the granules ordered and their status will be printed automatically as well as emailed to the address provided. Additional information on the order, including request URLs, can be viewed by setting the optional keyword input 'verbose' to True."
+    "Then, we can send the order to NSIDC using the order_granules function. Information about the granules ordered and their status will be printed automatically. Status information can also be emailed to the address provided when the `email` kwarg is set to `True`. Additional information on the order, including request URLs, can be viewed by setting the optional keyword input 'verbose' to True."
```
Comment (weiji14) [inline examples/ICESat-2_DAAC_DataAccess_Example.ipynb:430]: Could you also modify Line 439 below to say `# region_a.order_granules(verbose=True, subset=False, email=False)`? Just so users looking at the Jupyter notebook know how to get the email notifications still.

**Review by weiji14 — COMMENTED** [review]
Thanks @trey-stafford! Just one suggestion on the jupyter notebook. I'll let @JessicaS11 have the final review and say on this.

P.S. I've fixed the unrelated test failure at #241.

File: examples/ICESat-2_DAAC_DataAccess_Example.ipynb (hunk 1 above)
Comment (trey-stafford) [reply examples/ICESat-2_DAAC_DataAccess_Example.ipynb:430]: 4a5b393

**Comment by JessicaS11** [comment]
@trey-stafford Thanks for this update from NSIDC! If you update the base with the test fix, I'll approve the PR so the checks run and we can merge this PR.

**Comment by trey-stafford** [comment]
> @trey-stafford Thanks for this update from NSIDC! If you update the base with the test fix, I'll approve the PR so the checks run and we can merge this PR.

@JessicaS11 f2f882e updates this branch with `development`, tests should pass!

**Review by weiji14 — APPROVED** [review]

**Pull request merged**